receptor_affinity/mesh.py

```diff
@@ -427,6 +416,36 @@ def ping(self, count=10):
             results[node.name] = node.ping(count)
         return results
 
+    def settle(self):
```

Comment:
Is there a reason not to be consistent and mark the return here too?

Reply:
That's my mistake. I'll add one.